### PR TITLE
Added mentor info (Deepjyoti Mondal); Geolocation !!

### PR DESCRIPTION
**Pull request by yashk0thari**
Added image of Mentor !!

Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -266,3 +266,14 @@
...
   lat: 25.4275
   lng: 81.7728
+  
+  
+- name: Deepjyoti Mondal
+  github: djmgit
+  twitter: djm_dev96 
+  image: Mentor.jpeg
+  
+  Geolocation (city or country: Kolkata)
```
Comment (anshumanv) [inline _data/mentors.yml]: This line in not needed.

Comment (yashk0thari) [reply _data/mentors.yml]: ohk; so should i remove it ??

Comment (anshumanv) [reply _data/mentors.yml]: yes, remove line 276

Comment (yashk0thari) [reply _data/mentors.yml]: kk

File: _data/mentors.yml [hunk 2]
```diff
@@ -266,3 +266,12 @@
   image: Ayush.jpeg
   lat: 25.4275
   lng: 81.7728
+  
+  
+- name: Deepjyoti Mondal
+  github: djmgit
+  twitter: djm_dev96 
+  image: Mentor.jpeg
```
Comment (anshumanv) [inline _data/mentors.yml:274]: Good going, please can you rename the file to something more sensible? By his name would be preferable.

Comment (yashk0thari) [reply _data/mentors.yml:274]: ohkk

Comment (anshumanv) [reply _data/mentors.yml:274]: Thanks, and good work.

**Comment by yashk0thari** [comment]
made necessary changes; squashing them now

**Comment by anshumanv** [comment]
You haven't renamed the file.

**Comment by yashk0thari** [comment]
i have; its Deepjyoti now

**Comment by yashk0thari** [comment]
Did you mean for the PR ??

File: _data/mentors.yml (hunk 2 above)
Comment (anshumanv) [reply _data/mentors.yml:274]: image still says - `Mentor.jpeg`

**Comment by yashk0thari** [comment]
ohh the image; lol; wait...

**Comment by yashk0thari** [comment]
i'm making a new PR; this is getting too unorganised so yeah

**Pull request closed without merging**